src/AnsiUtils.ps1

```diff
@@ -64,6 +64,11 @@ function Get-VirtualTerminalSequence ($color, [int]$offset = 0) {
         return "${AnsiEscape}$(38 + $offset);2;${r};${g};${b}m"
     }
 
+    # Force 'DarkYellow' to ConsoleColor, since it is not an HTML color
+    if ($color -eq [ConsoleColor]::DarkYellow) {
```

Comment:
Recommend using `[System.ConsoleColor]` here and on the next line. Except for the native types `[string]`, we've tried to specify the full type name so PS doesn't use its "look twice" for the type algorithm.